project/spotify/static/spotify/neighbors.js

```diff
@@ -0,0 +1,16 @@
+$(document).ready(function() {
+	
+	$.getJSON("/neighborsinfo",function(data) {
+		var users = data['user_list']
+		var neighbors = data['neighbors_list']
```

Comment:
Mixed double and single quotes.<br>['neighbors_list'] is better written in dot notation.<br>Missing semicolon.